components/pages/UserProfile/UserProfile.module.css

```diff
@@ -162,9 +162,10 @@
   display: flex;
 }
 .postsSection {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 293px 293px 293px;
```

Comment:
You can change it back to flex so it works on mobiles.

Reply:
Done.